soc/arm/nxp_kinetis/kwx/Kconfig.defconfig.mkw41z4

```diff
@@ -107,4 +107,30 @@ config NET_CONFIG_IEEE802154_DEV_NAME
 
 endif # NETWORKING
 
+#
+# Platform dependent cryptographically secure random number support.
```

Comment:
This description would seem to belong in subsys/random/Kconfig instead of here, or?